ERROR [com.zaxxer.hikari.pool.HikariPool] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.springframework.boot.autoconfigure.orm.jpa.DatabaseLookup] at org.springframework.jdbc.support.JdbcUtils.extractDatabaseMetaData(JdbcUtils.java:328): Could not get Connection for extracting meta-data; nested exception is org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLSyntaxErrorException: Access denied for user 'acme-user'@'%' to database 'starter-project'
at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:81): Failed to obtain JDBC Connection; nested exception is java.sql.SQLSyntaxErrorException: Access denied for user 'acme-user'@'%' to database 'starter-project'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
ERROR [com.zaxxer.hikari.pool.HikariPool] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator] HHH000342: Could not obtain connection to query metadata : Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1778): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:275): Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.engine.jdbc.dialect.internal.DialectFactoryImpl.determineDialect(DialectFactoryImpl.java:100): Access to DialectResolutionInfo cannot be null when 'hibernate.dialect' not set
ERROR [com.zaxxer.hikari.pool.HikariPool] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.springframework.boot.autoconfigure.orm.jpa.DatabaseLookup] at org.springframework.jdbc.support.JdbcUtils.extractDatabaseMetaData(JdbcUtils.java:328): Could not get Connection for extracting meta-data; nested exception is org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLSyntaxErrorException: Access denied for user 'acme-user'@'%' to database 'starter-project'
at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:81): Failed to obtain JDBC Connection; nested exception is java.sql.SQLSyntaxErrorException: Access denied for user 'acme-user'@'%' to database 'starter-project'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
ERROR [com.zaxxer.hikari.pool.HikariPool] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator] HHH000342: Could not obtain connection to query metadata : Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1778): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:275): Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.engine.jdbc.dialect.internal.DialectFactoryImpl.determineDialect(DialectFactoryImpl.java:100): Access to DialectResolutionInfo cannot be null when 'hibernate.dialect' not set
WARN [com.zaxxer.hikari.pool.HikariPool] {} - Thread starvation or clock leap detected (housekeeper delta={}).
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [hernandez_bulletin]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1778): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [hernandez_bulletin]
at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:402): [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [hernandez_bulletin]
at org.hibernate.tool.schema.internal.AbstractSchemaValidator.validateTable(AbstractSchemaValidator.java:121): Schema-validation: missing table [hernandez_bulletin]
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.MappingException: Could not determine type for: java.util.Collection, at table: notice, for columns: [org.hibernate.mapping.Column(related_notices)]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1778): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.MappingException: Could not determine type for: java.util.Collection, at table: notice, for columns: [org.hibernate.mapping.Column(related_notices)]
at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:402): [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.MappingException: Could not determine type for: java.util.Collection, at table: notice, for columns: [org.hibernate.mapping.Column(related_notices)]
at org.hibernate.mapping.SimpleValue.getType(SimpleValue.java:486): Could not determine type for: java.util.Collection, at table: notice, for columns: [org.hibernate.mapping.Column(related_notices)]
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: acme.framework.helpers.Assert.state(Assert.java:40): End-point no disponible
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: java.lang.AssertionError
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../anonymous/notice/form.jsp'.
at org.apache.jsp.tag.web.print_tag.doTag(print_tag.java:168): org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../anonymous/notice/form.jsp'.
at org.apache.jsp.tag.web.print_tag.doTag(print_tag.java:168): org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../anonymous/notice/form.jsp'.
at org.apache.jsp.tag.web.print_tag.doTag(print_tag.java:168): org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../anonymous/notice/form.jsp'.
at org.apache.jsp.tag.web.print_tag.doTag(print_tag.java:168): org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at org.hibernate.collection.internal.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:602): failed to lazily initialize a collection of role: acme.entities.notices.Notice.relatedNotices, could not initialize proxy - no Session
WARN [com.zaxxer.hikari.pool.HikariPool] {} - Thread starvation or clock leap detected (housekeeper delta={}).
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: acme.framework.helpers.Assert.state(Assert.java:40): Access is not authorised
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: acme.framework.helpers.Assert.state(Assert.java:40): Acceso no autorizado
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../authenticated/technology-record/list.jsp'.
at org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/list.tag(82,6) '${JspHelper.computeDataText(requestScope[index_path], format)}' java.lang.ClassCastException: acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../authenticated/tool-record/list.jsp'.
at org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/list.tag(82,6) '${JspHelper.computeDataText(requestScope[index_path], format)}' java.lang.ClassCastException: acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../authenticated/technology-record/list.jsp'.
at org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/list.tag(82,6) '${JspHelper.computeDataText(requestScope[index_path], format)}' java.lang.ClassCastException: acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../authenticated/technology-record/list.jsp'.
at org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/list.tag(82,6) '${JspHelper.computeDataText(requestScope[index_path], format)}' java.lang.ClassCastException: acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../authenticated/technology-record/list.jsp'.
at org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/list.tag(82,6) '${JspHelper.computeDataText(requestScope[index_path], format)}' java.lang.ClassCastException: acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[jsp]] at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): JSPException including path '../authenticated/technology-record/list.jsp'.
at org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/list.tag(82,6) '${JspHelper.computeDataText(requestScope[index_path], format)}' java.lang.ClassCastException: acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/acme-incubator].[dispatcherServlet]] at java.text.MessageFormat.format(MessageFormat.java:865): acme.entities.sectors.Sector cannot be cast to [Ljava.lang.Object;
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] HTTP GET /acme-incubator/META-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.META_002dINF.views.framework.panic_jsp] Exception: acme.framework.helpers.Assert.state(Assert.java:40): Access is not authorised
